### PR TITLE
Adds conf-radare2

### PR DESCRIPTION
**Pull request by gitoleg**
This PR adds `conf-radare2` package
See https://github.com/BinaryAnalysisPlatform/bap/pull/1195 for more information

Fixes https://github.com/fxfactorial/ocaml-radare2/issues/7

### CODE REVIEW

**Comment by ivg** [comment]
The failures are due to missing radare2 executable on some distributions. Also looping in @XVilka for the situational awareness.

**Comment by XVilka** [comment]
Yes, already subscribed. I will use it as a dependency for "radare2"
package as well once merged.

**Comment by ivg** [comment]
@kit-ty-kate, a friendly ping :) Could you guys merge this, we have two issues and two PR in two different repositories blocked on it :)

File: packages/conf-radare2/conf-radare2.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+]
+depends: [
+   "ocaml"
+   "base-unix"
+   "conf-which" {build}
+]
+substs: [ "find-radare2.ml" ]
+flags: [ conf ]
+
+depexts: [ "radare2" ]
```
Comment (kit-ty-kate) [inline packages/conf-radare2/conf-radare2.0.1/opam:21]: is the name of the package `radare2` on **every single** distributions?
Could it be replaced by:
```
depexts: [
  ["radare2"] {os-family = "debian"}
  ["radare2"] {os-distribution = "fedora"}
  ["radare2"] {os-distribution = "arch"}
  ["radare2"] {os-distribution = "gentoo"}
  ["radare2"] {os-distribution = "homebrew" & os = "macos"}
]
```
(taken from the `radare2` package) instead?

File: packages/conf-radare2/conf-radare2.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["ocaml" "unix.cma" "find-radare2.ml"]
+]
+depends: [
+   "ocaml"
```
Comment (kit-ty-kate) [inline packages/conf-radare2/conf-radare2.0.1/opam:14]: if I remember correctly the `{| .. |}` type of string came in OCaml 4.02 so I think this should be constrained.

File: packages/conf-radare2/conf-radare2.0.1/opam (hunk 1 above)
Comment (gitoleg) [reply packages/conf-radare2/conf-radare2.0.1/opam]: done!

File: packages/conf-radare2/conf-radare2.0.1/opam (hunk 2 above)
Comment (gitoleg) [reply packages/conf-radare2/conf-radare2.0.1/opam]: done! :)

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**